cli/types/index.d.ts

```diff
@@ -756,7 +756,7 @@ declare namespace Cypress {
      *
      * @see https://on.cypress.io/scrollintoview
      */
-    scrollIntoView(options?: Partial<ScrollIntoViewOptions>): Chainable<Subject>
+    scrollIntoView(options?: Partial<ScrollToOptions>): Chainable<Subject>

```

Comment:
Ah, oops. `ScrollIntoViewOptions` is defined in `lib.dom.d.ts`. According to the docs, there is a different set of options for `scrollTo` and `scrollIntoView`. `scrollTo` doesn't have `offset`. I think the correctly implementation would be to define a `ScrollIntoViewOptions` interface that extends `ScrollToOptions` and adds the `offset` property.